client/src/pages/Login.js

```diff
@@ -0,0 +1,56 @@
+import logoBike from "./logoBike.svg";
+import Form from "react-bootstrap/Form";
+
+
+const Login = () => {
+	return (
+		<form>
+			<div className="Login-page">
+				<div className="flex-container">
```

Comment:
It looks like the `flex-container` class can be on the `form` instead. Can we delete this `div` and move the `flex-container` class to `form`?

Reply:
yes the code  will look  more tidy